lib/person.rb

```diff
@@ -1 +1,71 @@
-# your code goes here
+class Person
+  attr_reader :name
+  attr_accessor :bank_account, :happiness, :hygiene
+
+  def initialize(name, bank_account=25, happiness=8, hygiene=8)
+    @name = name
+    @bank_account = bank_account
+    @happiness = happiness
+    @hygiene = hygiene
+  end
+
+  def happiness
+    @happiness
+  end
+
+  def happiness=(value)
+    @happiness = value.clamp(0, 10)
+  end
+
+  def hygiene
+    @hygiene
+  end
```

Comment:
Since you have `attr_accessor`s, you don't need to define the happiness and hygiene reader methods:
```suggestion
  def happiness=(value)
    happiness = value.clamp(0, 10)
  end
```
And same throughout.